metadata.py

```
@@ -520,7 +522,7 @@ def build_dict(path: str) -> dict:
         month = parse_month(path)
         year = parse_year(path)
         if month and year:
-            dic["release_date"] = f"{month}/year"   
+            dic["release_date"] = f"{month}/year"
         else:
             raise ValueError("Missing month/year")
```

Ruff (0.11.9)

Abstract `raise` to an inner function

(TRY301)